2021-11-24 22:31:50,897 INFO    MainThread:43599 [wandb_setup.py:_flush():71] setting env: {}
2021-11-24 22:31:50,897 INFO    MainThread:43599 [wandb_setup.py:_flush():71] setting login settings: {'anonymous': 'must'}
2021-11-24 22:31:50,897 INFO    MainThread:43599 [wandb_init.py:_log_setup():371] Logging user logs to /home/snamburi/Debarshi/wandb/run-20211124_223150-3v3qlj7a/logs/debug.log
2021-11-24 22:31:50,897 INFO    MainThread:43599 [wandb_init.py:_log_setup():372] Logging internal logs to /home/snamburi/Debarshi/wandb/run-20211124_223150-3v3qlj7a/logs/debug-internal.log
2021-11-24 22:31:50,897 INFO    MainThread:43599 [wandb_init.py:init():404] calling init triggers
2021-11-24 22:31:50,897 INFO    MainThread:43599 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-11-24 22:31:50,897 INFO    MainThread:43599 [wandb_init.py:init():449] starting backend
2021-11-24 22:31:50,898 INFO    MainThread:43599 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-24 22:31:50,929 INFO    MainThread:43599 [backend.py:ensure_launched():199] starting backend process...
2021-11-24 22:31:50,950 INFO    MainThread:43599 [backend.py:ensure_launched():204] started backend process with pid: 43611
2021-11-24 22:31:50,953 INFO    MainThread:43599 [wandb_init.py:init():458] backend started and connected
2021-11-24 22:31:50,955 INFO    MainThread:43599 [wandb_init.py:init():519] updated telemetry
2021-11-24 22:31:50,958 INFO    MainThread:43599 [wandb_init.py:init():549] communicating current version
2021-11-24 22:31:51,746 INFO    MainThread:43599 [wandb_init.py:init():554] got version response 
2021-11-24 22:31:51,746 INFO    MainThread:43599 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-24 22:31:51,943 INFO    MainThread:43599 [wandb_init.py:init():592] starting run threads in backend
2021-11-24 22:31:53,417 INFO    MainThread:43599 [wandb_run.py:_console_start():1816] atexit reg
2021-11-24 22:31:53,418 INFO    MainThread:43599 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-24 22:31:53,418 INFO    MainThread:43599 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-24 22:31:53,422 INFO    MainThread:43599 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-24 22:31:53,422 INFO    MainThread:43599 [wandb_init.py:init():619] run started, returning control to user process
2021-11-24 22:31:53,422 INFO    MainThread:43599 [wandb_run.py:_config_callback():962] config_cb None None {'epochs': 5, 'batch_size': 1, 'learning_rate': 1e-05, 'val_percent': 0.1, 'save_checkpoint': True, 'img_scale': 0.5, 'amp': True}
2021-11-24 22:31:53,588 INFO    MainThread:43599 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-11-24 22:31:53,588 INFO    MainThread:43599 [wandb_run.py:_restore():1758] restore
2021-11-24 22:31:55,938 INFO    MainThread:43599 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 564
}

2021-11-24 22:31:56,107 INFO    MainThread:43599 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 564
}

2021-11-24 22:31:56,987 INFO    MainThread:43599 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 9395
}

2021-11-24 22:31:57,093 INFO    MainThread:43599 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 9395
}

2021-11-24 22:31:57,196 INFO    MainThread:43599 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9395
  total_bytes: 9395
}

2021-11-24 22:31:57,299 INFO    MainThread:43599 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9395
  total_bytes: 9395
}

2021-11-24 22:31:57,402 INFO    MainThread:43599 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9395
  total_bytes: 9395
}

2021-11-24 22:31:57,505 INFO    MainThread:43599 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9395
  total_bytes: 9395
}

2021-11-24 22:31:57,747 INFO    MainThread:43599 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9395
  total_bytes: 9395
}

2021-11-24 22:31:58,015 INFO    MainThread:43599 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9395
  total_bytes: 9395
}
local_info {
}

2021-11-24 22:31:59,190 INFO    MainThread:43599 [wandb_run.py:_append_files():2184] logging synced files
